### PR TITLE
Monit

### PR DESCRIPTION
**Pull request by spikeeSakshu**
Please do a SEO check for my blog

### CODE REVIEW

File: source/_posts/To-monitor-and-restart-services-using-Monit.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/To-monitor-and-restart-services-using-Monit.md:1]: Nayan is a platform that offer :  this line is repeated in the bottom so please remove it from there.

Comment (Rajeev111974) [inline source/_posts/To-monitor-and-restart-services-using-Monit.md:1]: Image should be optimized with Alt text

Comment (Rajeev111974) [inline source/_posts/To-monitor-and-restart-services-using-Monit.md:1]: Targeted url :  targeted keyword is there in url

**Comment by Rajeev111974** [comment]
Dear ‘Saksham,

 

We have reviewed the blog and mentioned some comment which require changes in the blog, please check and update it for further process

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/To-monitor-and-restart-services-using-Monit.md (hunk 1 above)
Comment (spikeeSakshu) [reply source/_posts/To-monitor-and-restart-services-using-Monit.md:1]: Sir it is not repeated. Can you please check it.

Comment (spikeeSakshu) [reply source/_posts/To-monitor-and-restart-services-using-Monit.md:1]: I have used the asset_image which has the last component as title that is an alternative to alt_text

Comment (spikeeSakshu) [reply source/_posts/To-monitor-and-restart-services-using-Monit.md:1]: I dont know what to do here. Can you please help me out?

**Comment by Rajeev111974** [comment]
Dear saksham,

 

Please check this, repeating line 2 times here.

 



 

 

 

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/To-monitor-and-restart-services-using-Monit.md (hunk 1 above)
Comment (spikeeSakshu) [reply source/_posts/To-monitor-and-restart-services-using-Monit.md:1]: Sir, I have resolved this issue

Comment (Rajeev111974) [reply source/_posts/To-monitor-and-restart-services-using-Monit.md:1]: Done

**Review by Rajeev111974 — APPROVED** [review]

**Pull request merged**